packages/design-system/src/components/Button/Button.md

```diff
@@ -179,6 +179,47 @@ Button expanded to the full width to fill the parent container (max-width: 320px
 </ComponentHtmlMarkup>
 ```
 
+### Loading with label
+
+```js
+  initialState = { loading: true };
+
+  toggleLoading = () => {
```

Comment:
I think you could use already existing section `### Loading` and add only one example to show that it's possible to use a label for loader